packages/auth/demo/src/index.js

```diff
@@ -49,8 +49,8 @@ import {
   signInWithCredential,
   signInWithCustomToken,
   signInWithEmailAndPassword,
+  signInWithEmailLink,
   TotpMultiFactorGenerator,
-  TotpSecret,
```

Comment:
This seems to be the only `Totp` change int his PR. Is its removal on purpose?

Reply:
Yes, the removal is intentional because it's unused.